.github/workflows/release.yml

```diff
@@ -20,10 +20,10 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 20
 
       - name: Update npm
-        run: npm i -g npm@7
+        run: npm i -g npm@10
```

Comment:
would honestly rec removing this and just using the version of npm that comes with the version of node